CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 tomland uses [PVP Versioning][1].
 The changelog is available [on GitHub][2].
 
+## 1.1.1.0
```

Comment:
Let's do a minor version bump and today's date. I will release it to Hackage after this PR is merged
```suggestion
## 1.1.0.1 — Jul 10, 2019
```